crates/pop-cli/src/commands/new/parachain.rs

```diff
@@ -55,58 +45,125 @@ pub struct NewParachainCommand {
+				};
+
+				is_template_supported(provider, &template)?;
+				let config = get_customization_value(
+					&template,
+					self.symbol.clone(),
+					self.decimals,
+					self.initial_endowment.clone(),
+				)?;
+
+				generate_parachain_from_template(name, provider, &template, None, config)
 			},
-		)?;
-		if let Err(err) = Git::git_init(destination_path.as_path(), "initialized parachain") {
-			if err.class() == git2::ErrorClass::Config && err.code() == git2::ErrorCode::NotFound {
-				outro_cancel("git signature could not be found. Please configure your git config with your name and email")?;
-			}
+		};
+	}
+}
+
+fn generate_parachain_from_template(
+	name_template: &String,
+	provider: &Provider,
+	template: &Template,
+	tag_version: Option<String>,
+	config: Config,
+) -> Result<()> {
+	intro(format!(
+		"{}: Generating \"{}\" using {:?} from {:?}!",
+		style(" Pop CLI ").black().on_magenta(),
+		name_template,
+		template,
+		provider
+	))?;
+	let destination_path = check_destination_path(name_template)?;
+
+	let mut spinner = cliclack::spinner();
+	spinner.start("Generating parachain...");
+	let tag = instantiate_template_dir(template, destination_path, tag_version, config)?;
+	if let Err(err) = Git::git_init(destination_path, "initialized parachain") {
+		if err.class() == git2::ErrorClass::Config && err.code() == git2::ErrorCode::NotFound {
+			outro_cancel("git signature could not be found. Please configure your git config with your name and email")?;
 		}
-		spinner.stop("Generation complete");
-		if let Some(tag) = tag {
-			log::info(format!("Version: {}", tag))?;
+	}
+	spinner.stop("Generation complete");
+	if let Some(tag) = tag {
+		log::info(format!("Version: {}", tag))?;
+	}
+
+	cliclack::note(
+			"NOTE: the resulting parachain is not guaranteed to be audited or reviewed for security vulnerabilities.",
+		format!("Please consult the source repository at {} to assess production suitability and licensing restrictions.", template.repository_url()?))?;
+
+	outro(format!("cd into \"{}\" and enjoy hacking! 🚀", name_template))?;
+
+	Ok(())
+}
+
+async fn guide_user_to_generate_parachain() -> Result<()> {
+	intro(format!("{}: Generate a parachain", style(" Pop CLI ").black().on_magenta()))?;
+
+	let mut prompt = cliclack::select("Select a template provider: ".to_string());
+	for (i, provider) in Provider::providers().iter().enumerate() {
+		if i == 0 {
+			prompt = prompt.initial_value(provider);
 		}
-		outro(format!("cd into \"{}\" and enjoy hacking! 🚀", destination_path.display()))?;
-		Ok(())
+		prompt = prompt.item(
+			provider,
+			provider.name(),
+			format!(
+				"{} {} available option(s)",
+				provider.description(),
+				provider.templates().len()
+			),
+		);
 	}
+	let provider = prompt.interact()?;
+	let template = display_select_options(provider)?;
+
+	let url = url::Url::parse(&["https://github.com/", template.repository_url()?].concat())
```

Comment:
Can probably just move the `https://github.com/` prefix into the repository string to eliminate this complexity.

Reply:
Yes, good idea! Added in the Template enum the two methods to return the url needed, so will make mantaining it easier